### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by Kidd254**
The finish mobile version pull request is an extension of the previous pull request (setup and mobile first). In this pull request, added content to the  HTML and CSS files. I also added several img files. I also made several changes by modifying the HTML and CSS files so that the output matches the Figma design. I used es linter tool to make the necessary changes to ensure that the CSS file format aligns with the popular style.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @Kidd254,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly use this template and update all the sections according to your project. You can use find the template on this [link](https://github.com/microverseinc/readme-template#built-with). Your section headings are not there and the links on the top are not working. Also, kindly add a link to your license file.
- Please fix the PR description and try to be more descriptive it needs to include what changes were made on this branch.
- Please use a more descriptive commit description for your next commits. So you can keep your project history in a more professional way.
Screenshot 2022-12-01 at 11 07 48 PM

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -40,7 +40,147 @@ <h2 class="social order">LET'S CONNECT</h2>
...
+                    <div class="li11">
+                        <article class="text1">html</article>
+                    </div>
+                    <div class="li22">
+                        <article class="text2">css</article>
+                    </div>
+                    <div class="li33">
+                        <article class="text3">javascript
+                    </div>
+                </div>
```
Comment (MaganaAsiati) [inline index.html]: - You did a great job by making the list of technology in the ` div ` however I can suggest for the best practices to use  `<ul> and  <li>`  tags  when you structure your list so The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li :+1:

File: index.html [hunk 2]
```diff
@@ -40,7 +40,147 @@ <h2 class="social order">LET'S CONNECT</h2>
...
```
Comment (MaganaAsiati) [inline index.html]: - You did a great job by making the list of technology in the ` div ` however I can suggest for the best practices to use  `<ul> and  <li>`  tags  when you structure your list so The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li :+1:

File: index.html [hunk 3]
```diff
@@ -40,7 +40,147 @@ <h2 class="social order">LET'S CONNECT</h2>
...
+                <div class="action">
```
Comment (MaganaAsiati) [inline index.html]: - You did a great job by making the list of technology in the ` div ` however I can suggest for the best practices to use  `<ul> and  <li>`  tags  when you structure your list so The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li :+1:

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @Kidd254 , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

- You did a great job on your design it looks nice however the are some issues you need to fix so that it will make your design stick to the Figma design as required in the project  :+1: :grinning: :grinning: :grinning: 
  - Kindly make  your design  responsive by removing those spaces around the body  of your page for doing that I can suggest you not use the fix for the main container of your project, and make  your page take the max screen at 100%
  - Kindly add also the about section on your page as it is also part of the requirement :+1: 
  - Please implement the button hover styles for all buttons on your site. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements pray
    
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Kidd254** [comment]
hello @MaganaAsiati, about (making your design responsive by removing those spaces around the body of your page for doing that I can suggest you not use the fix for the main container of your project, and make your page take the max screen at 100%) we are yet to cover responsive web design...

**Review by frank1738 — APPROVED** [review]
Hi @Kidd254 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good design 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**